cumulus_library/study_parser.py

```diff
@@ -427,6 +445,9 @@ def export_study(self, cursor: object, data_path: PosixPath) -> List:
             dataframe = cursor.execute(query).as_pandas()
             path = Path(f"{str(data_path)}/{self.get_study_prefix()}/")
             path.mkdir(parents=True, exist_ok=True)
+            dataframe = dataframe.sort_values(
+                by=list(dataframe.columns), ascending=False, na_position="first"
+            )
```

Comment:
After seeing the churn in the reference data files, I threw this in, which gets us back to the original format for exports before we removed the ORDER BY clauses for performance reasons.